src/erlastic_search.erl

```diff
@@ -91,11 +91,15 @@
         ,reindex/2
         ,aliases/1
         ,aliases/2
+        ,bulk_operation/1
+        ,bulk_operation/2
 ]).
 
 
 -include("erlastic_search.hrl").
 
+-type operations() :: [{index | create | update, binary(), binary(), binary(), json()} | {delete, binary(), binary(), binary()}].
```

Comment:
Would make more sense to define this type right before the only funs where it's used. And would be good to create named, inline types (`Index`, `Type`, etc...) instead of those generic `binary()`.

Plus I don't think it's accurate, since an `operation()` can also comprise additional header info.